transformations/pinyin/requirements.txt

```diff
@@ -0,0 +1,2 @@
+g2pM==0.1.2.5
+zh-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/zh_core_web_sm-3.0.0/zh_core_web_sm-3.0.0.tar.gz
```

Comment:
I think it would be heavy for this to be downloaded by default for everyone. So, I wouldn't suggest to be put like this. 
Temporarily, you can rename this requirements.txt to requirements-disable.txt.
And add heavy=True in the implementation. 
@Nickeilf @AbinayaM02 @aadesh11

Reply:
What does "temporarily" mean in this context? Do you want me to make this change, and then undo it before merging the PR?

Reply:
No. you can make this change and let it be merged. It just won't install and build by default. You can mention in the README  the procedure to additionally setup this transformation so that users would know that they would need to install this requirements.txt additionally.